tools/serverpod_cli/lib/src/generator/types.dart

```diff
@@ -266,15 +266,18 @@ class TypeDefinition {
     return 'ColumnSerializable';
   }
 
-  /// Strip the outer most future of this type.
-  /// Throws, if this type is not a future.
-  TypeDefinition stripFuture() {
+  /// Strip the outer most Future or Stream of this type.
+  /// Throws, if this type is not a Future or Stream.
+  TypeDefinition stripFutureOrStream() {
```

Comment:
Suggestion: `retrieveGenericType` instead? This is more general and means the same thing.

Reply:
we could also change the implementation do do that too I suppose.

Reply:
Good suggestion! 👍